src/Serilog/Events/MessageTemplate.cs

```diff
@@ -21,11 +21,6 @@ namespace Serilog.Events;
 /// </summary>
 public class MessageTemplate
 {
-    /// <summary>
-    /// Represents the empty message template.
-    /// </summary>
-    public static MessageTemplate Empty { get; } = new(Enumerable.Empty<MessageTemplateToken>());
```

Comment:
This one's a public API; I don't think we have a good reason to remove it right now.

Reply:
@nblumhardt just to play devils advocate... what is the use case?